includes/class-form-entry-manager.php

```diff
@@ -51,4 +51,54 @@ public function all() {
     public function get( $entry_id ) {
         return new WeForms_Form_Entry( $entry_id, $this->form );
     }
+
+    /**
+     * Format Entry Value.
+     *
+     * @static

```

Comment:
Just a comment, you should usually put the @static tag last. It's not a requirement, but it is customary. From what I've seen anyways.